client/src/actions/team.js

```diff
@@ -277,17 +278,40 @@ function teamDataAllocator(teamData) {
  *
  * @param {*} teamId id of the team
  * @param {*} teamData data object of the data to be patched
+ * @param {*} successMsg the success message to display if different from default one
  * @returns
  */
-export const updateTeam = (teamId, teamData) => async (dispatch) => {
+export const updateTeam = (teamId, teamData, successMsg = 'Team has been updated') => async (dispatch) => {
```

Comment:
Out of curiosity , Isn't the successMessageCreator dispatched here like the only success toast being created ?  if so, having it like line 290 would be the best practise

Reply:
its there to allow you to optionally change the msg given, depending on the update action you do, and a default one so that its backwards compatible